sisyphus/job_path.py

```diff
@@ -363,6 +374,8 @@ def is_set(self):
                                                           < gs.CACHE_FINISHED_RESULTS_MAX_SIZE))
     def get(self):
         if not self.is_set():
+            if gs.RAISE_VARIABLE_NOT_SET_EXCEPTION:
```

Comment:
Why is it an error to access a variable that has a backup?

Reply:
I don't really like the solution of having a backup value at this point since it doesn't allow to return `None`. The recently add `fallback` method solves this in a cleaner way. So I plan to remove the possibility to add a backup directly to the Variable at some point.

Reply:
In the ASR recipes it's not used. Is backup used in any of the MT recipes? If not you could remove it directly.

Reply:
It's used inside the MT recipes, so I would like to keep both for a while for backwards compatibility during the conversion.